src/Serilog.Sinks.EventLog/LoggerConfigurationEventLogExtensions.cs

```diff
@@ -110,12 +103,11 @@ public static class LoggerConfigurationEventLogExtensions
         LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
         IEventIdProvider? eventIdProvider = null)
     {
-#if FEATURE_RUNTIMEINFORMATION
         if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
Should it be using !OperatingSystem.IsWindows() where available? (could possibly be better for trimming on not-windows platforms?)

Reply:
Indeed, just addressed in f114631e8329a280f3d00bebba0705e891a05c75.

I did not notice it would not even compile on `net462` because I was on macOS. With this last commit, the build on AppVeyor should pass now.